client/src/test/scala/org/http4s/client/middleware/FollowRedirectSpec.scala

```diff
@@ -75,7 +75,7 @@ class FollowRedirectSpec
       ) = {
         val u = uri("http://localhost") / status.code.toString
         val req: Request[IO] = method match {
-          case _: Method.PermitsBody if body.nonEmpty =>
+          case (OPTIONS | PATCH | POST | PUT) if body.nonEmpty =>
```

Comment:
This could also be preserved as a property of `Method`, but has the same problem as cacheability: someone has to go through all the WebDAV specs.  The answer there is usually "Yes, but a specific type of XML."